tests/psimulate/worker/test_vivarium_work_horse.py

```diff
@@ -102,7 +54,13 @@ def test_get_backup(
                     "random_seed": [random_seed],
                     "job_id": job_id,
                     "branch_key": ["branch_value"],
-                }
+                },
+                {
```

Comment:
I added a, let's say, test context here, to make sure that we don't pick up a pickle with the same branch configuration but a different draw or seed. I almost submitted a bug where that could happen.